### DISCUSSION TITLE
Configure in json

### DISCUSSION THREAD

**Discussion by godalming123** [hyprwm/Hyprland#476] [General]
Have you considered configuration in json it allows:
 - Less searching in the docs as in my opinion more people know json syntax then conf syntax
 - Easier parsing and edge-case handling as more standard libraries for reading json exist

**Comment by fufexan** [comment]
JSON is probably a worse idea than TOML, imo. But I believe Vaxry said once that he doesn't want a 3rd party lib to take care of as well.

**Reply by godalming123** [reply]
Just an idea

**Comment by vaxerski** [comment]
no

**Comment by adrian-gierakowski** [comment]
Is there a tool\lib for converting hyperland config to json and back?
